src/fmu/dataio/dataio.py

```diff
@@ -78,24 +78,40 @@ def _validate_variable(key, value, legals) -> bool:
     return True
 
 
-def _check_global_config(globalconfig: dict, strict: bool = True):
+def _check_global_config(
+    globalconfig: dict, strict: bool = True, action: str = "error"
+) -> bool:
     """A check/validation of static global_config.
 
-    Currently not a full validation. For now, just check that some required
-    keys are present in the config and raise if not.
+    Currently far from a full validation. For now, just check that some required
+    keys are present in the config and warn/raise if not.
     """
 
     if not globalconfig and not strict:
-        warn(
-            "Empty global config, expect input from environment_variable instead",
-            UserWarning,
+        logger.info(
+            "Empty global config, expect input from environment_variable insteac"
         )
-        return
+        return False
 
     config_required_keys = ["access", "masterdata", "model"]
+    flag = True
     for required_key in config_required_keys:
         if required_key not in globalconfig:
-            raise ValidationError(f"Required key '{required_key}' not found in config.")
+            flag = False
+
+    if flag is False:
+        msg = (
+            "One or more required keys not found in config (perhaps config is empty)."
```

Comment:
Say which keys are required. User would like to know if he is not expecting the warning.